assets/panel/debugger.properties

```diff
@@ -92,9 +92,13 @@ ignoreExceptionsItem=Ignore exceptions
 # item shown when a user is adding a new breakpoint.
 pauseOnUncaughtExceptionsItem=Pause on uncaught exceptions
 
-# LOCALIZATION NOTE (pauseOnExceptionsItem): The pause on exceptions button description
+# LOCALIZATION NOTE (pauseOnExceptionsItem): The pause on exceptions checkbox description
 # when the debugger will pause on all exceptions.
-pauseOnExceptionsItem=Pause on all exceptions
+pauseOnExceptionsItem=Pause on Exceptions
+
+# LOCALIZATION NOTE (ignoreCaughExceptionsItem): The pause on exceptions checkbox description
+# when any caught exceptions will not be paused
```

Comment:
to be consistent with the note of "pauseOnExceptionsItem", this could read "when the debugger will not pause on any caught exception"